editor/src/components/canvas/commands/set-css-length-command.ts

```diff
@@ -81,7 +85,11 @@ export const runSetCssLengthProperty: CommandFunction<SetCssLengthProperty> = (
     editorState,
     left(`no target element was found at path ${EP.toString(command.target)}`),
     (_, element) => {
-      return getModifiableJSXAttributeAtPath(element.props, command.property)
+      if (isJSXElement(element)) {
```

Comment:
This also feels like there's a case for a function that encapsulates this logic.

Reply:
Added to backlog.